mubench.pipeline/task_runner.py

```diff
@@ -40,18 +37,6 @@ def run(self) -> None:
                         self.black_list.extend(response)
             task.end()
 
-    def check(self) -> None:
-        for task in self.tasks:
-            logger = logging.getLogger()
-            logger.info("Checking requirements for %s...", task.name)
-            self.__check_requirements(task, logger)
-
-    @staticmethod
-    def __check_requirements(task, logger):
-        if not are_satisfied(task.get_requirements(), logger):
```

Comment:
The `Task.get_requirements()` method can now be deleted from `Task` and all its subclasses.